### PR TITLE
config: Extend sashal/linus-next:linus-next builds

### PR DESCRIPTION
**Pull request by pawiecz**
Fixes: https://github.com/kernelci/kernelci-core/issues/2686

### CODE REVIEW

File: config/pipeline.yaml [hunk 1]
```diff
@@ -2308,6 +2325,7 @@ scheduler:
     rules:
       tree:
         - next:master
+        - sashal-next:linus-next
```
Comment (a-wai) [inline config/pipeline.yaml:2328]:
```suggestion
        - sashal-next
```

Minor nit, we only build the `linux-next` branch for this tree. Keeping it as-is makes sense if we plan to build other branches in the future, though.

Comment (pawiecz) [reply config/pipeline.yaml]: Good catch, thanks! I'll squash it before merging this PR

**Review by a-wai — APPROVED** [review]

**Review by musamaanjum — APPROVED** [review]

**Comment by pawiecz** [comment]
I submitted two kernel build requests to verify proper alignment with linux-next build configuration:
- next@v6.11 (as control), Maestro TreeID: 9d00fd3c8bbe9dbca32978d9c757b7b2cdf2ad5c19d03f7cb2549098798731eb
- tip from the new tree (as of writing this comment), Maestro TreeID: 84d990b7eb8e0948f8152c05dbe4225f7741acbb93b181025858c2385d7fa5c3

After builds finish I'll compare triggered build sets to confirm they are aligned.

**Comment by pawiecz** [comment]
Verified with:

```
$ ./kci node find "treeid=9d00fd3c8bbe9dbca32978d9c757b7b2cdf2ad5c19d03f7cb2549098798731eb" "kind=kbuild" -l 100 | jq '.[] | .name' > control
$ ./kci node find "treeid=84d990b7eb8e0948f8152c05dbe4225f7741acbb93b181025858c2385d7fa5c3" "kind=kbuild" -l 100 | jq '.[] | .name' > new
$ diff -s control new
Files control and new are identical
```

**Comment by nuclearcat** [comment]
Thanks!

**Pull request merged**